src/sparse/KokkosSparse_spadd.hpp

```diff
@@ -310,6 +212,33 @@ namespace Experimental {
     CcolindsT ABpermAux;
   };
 
+  #ifdef KOKKOS_ENABLE_CUDA
```

Comment:
Why is this only on CUDA ?

Reply:
@srajama1 Because radix sort is the best on CPUs, and bitonic is the best on CUDA.This is just a template specialization of SortEntriesFunctor that chooses the best for the given execution space.